lib/skroutz/category.rb

```diff
@@ -1 +1,6 @@
-class Skroutz::Category < Skroutz::Resource; end
+class Skroutz::Category < Skroutz::Resource
+  association :skus
+  association :parent, class_name: 'Category'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.